common/windows-app-bootstrap.cpp

```diff
@@ -30,6 +32,13 @@ int CALLBACK WinMain(
     {
         std::wstring ws = szArgList.get()[i];
         std::string s(ws.begin(), ws.end());
+
+        if (s == rs2::metadata_helper::get_command_line_param())
```

Comment:
Wouldn't it be simpler if we made get_command_line_param() return a wstring?

Reply:
Conversion to string is for `args.push_back(s);` later. 
get_command_line_param doesn't utilize any special characters so I prefer to keep it simple